Prox/Prox/PlaceDetails/FilterViewController.swift

```diff
@@ -62,6 +70,7 @@ class FilterViewController: UIViewController {
             button.isSelected = enabledFilters[i]
             button.tag = i
             button.addTarget(self, action: #selector(didToggleFilter(sender:)), for: .touchUpInside)
+            button.heightAnchor.constraint(equalToConstant: 36)
```

Comment:
This constraint is unused. In order to activate it, you have to call either:
* `button.heightAnchor.constraint(equalToConstant: 36).isActive = true`
* `NSLayoutConstraint.activate([NSLayoutConstraint])`